### PR TITLE
Fix/issues 21112018

### PR DESCRIPTION
**Pull request by slavaleleka**
[//]: # (***You can delete or ignore strings starting with "[//]:" They will not be visible either way.)

***Description***:
* **Current behaviour**: 

[//]: # (Substitute this line with a description of the problem)

[//]: # (Replace %screenshot_url% below with a link to the screenshot of the problem. Also, you can paste image from clipboard instead. It will be automatically loaded.)
<details><summary>Screenshot:</summary>

image
</details><br/>

* **Expected behaviour**: 

[//]: # (Substitute this line with a description of what should happen normally. If needed, provide a screenshot below, same as above)

<details><summary>Screenshot:</summary>

image
</details><br/>

***Steps to reproduce the problem***:

[//]: # (Substitute this line with a step-by-step instruction how to reproduce the issue)

***System configuration***

**Filters:**

[//]: # (Substitute this line with the list of your active filters, separated by commas)

[//]: # (Please enter the correct values for your case to the table below)

Information                            | Value
---                                    | ---
Operating system:                      | ?
Operating system version (Android/iOS) | ?
Browser:                               | ?
AdGuard version:                       | ?
Filters enabled:                       | ?
AdGuard mode (Android only):           | VPN / Proxy (manual/auto)
Filtering quality (Android only):      | High-quality / High-speed / Simplified
HTTPS filtering (Android only):        | On (Default/Blacklist mode) / Off
Simplified filters (iOS only)          | On / Off
AdGuard DNS:                           | None / Default / Family Protection
Stealth mode options (Windows only)    | ?
Helpdesk ID (if exists):               | ?

[//]: # (This template is meant for missed ad/false positive reports, for other type of reports edit it accordingly)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)

### CODE REVIEW

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -138,6 +138,12 @@ forum.lowyat.net##.style_ad
 tonymacx86.com##.responsiveAdCentered
 u.gg###af-click > .af-left
 ||u.gg/affiliate/amazon/*.jpg
+!
+! https://github.com/AdguardTeam/AdguardFilters/issues/25637
+! SocialFilter turned off
```
Comment (ameshkov) [inline EnglishFilter/sections/specific.txt]: It does not matter, the user should enable social filter to hide such blocks

File: RussianFilter/sections/specific.txt [hunk 2]
```diff
@@ -16,11 +16,27 @@
 !+ NOT_PLATFORM(ios, ext_safari, ext_android_cb)
 Ly9hbi55YW5kZXgucnUv
 !
+vesti-ua.net## #dle-content > .mob2 > .hblock.cf.related_news_block:has(> section > noindex > script#MediaMetricsInject)
```
Comment (ameshkov) [inline RussianFilter/sections/specific.txt]: unnecessary extra space: `## #`

**Review by ameshkov — COMMENTED** [review]
Do not commit `.DS_Store`, it must be added to `.gitignore`

**Review by AdamWr — COMMENTED** [review]
Regarding `hotline.ua` - https://github.com/AdguardTeam/AdguardFilters/issues/19976
I'm not sure, but I think that this element should be hidden too:
<details><summary>Screenshot</summary>

hotline

</details>
<br>

Could you please move that rule - `emlakansiklopedisi.com##h1:has(a.habericilink)` https://github.com/AdguardTeam/AdguardFilters/blob/4c963c3f1e90cd8f55f618830786462d455cca7b/TurkishFilter/sections/specific.txt#L146-L147 to `css_extended.txt` section?

**Pull request closed without merging**